subsys/logging/Kconfig

```diff
@@ -424,6 +424,51 @@ config LOG_BACKEND_RTT_FORCE_PRINTK
 
 endif # LOG_BACKEND_RTT
 
+config LOG_BACKEND_SPINEL
+	bool "Enable Spinel protocol backend"
+	depends on !LOG_BACKEND_UART
```

Comment:
I think we shall also add dependency to OpenThread (`NET_L2_OPENTHREAD`).

Reply:
Done

Reply:
`depends on !LOG_BACKEND_UART`
isn't it possible to have spinel on different UART and console/log on another?

Reply:
Yes it is. I changed dependency to turn on Spinel backend only if UART backend is disabled or is enabled on another UART device than spinel. Is it ok now? I did some tests, using both backends at once and it seems to work properly.